spec/controllers/v1/damage_types_controller_spec.rb

```diff
@@ -0,0 +1,159 @@
+        submit
+        expect(Damage.count).to be 0
+      end
+      it 'delete the damage type' do
+        submit
+        expect(DamageType.count).to be 0
+      end
+    end
+    context 'damage type does not found' do
+      let(:damage_type) { double uuid: 0 }
+      it 'has a not found status' do
+        submit
+        expect(response).to have_http_status :not_found
+      end
+    end
+  end
+
+  describe 'GET #index' do
+    let(:submit) { get :index }
+    let(:damage_type) { create :damage_type }
+    let!(:damage1) { create :damage, damage_type: damage_type }
+    let!(:damage2) { create :damage, damage_type: damage_type }
+    it 'render all damgage type' do
+      submit
+      json = JSON.parse(response.body)
+      expect(json).to eql([{ 'uuid' => damage_type.uuid,
+                             'name' => damage_type.name,
+                             'damages' => [{ 'uuid' => damage1.uuid },
+                                           { 'uuid' => damage2.uuid }] }])
+    end
+  end
+
+  describe 'GET #show' do
+    let(:submit) { get :show, id: damage_type.uuid }
+    context 'damage_type found' do
+      let(:damage_type) { create :damage_type }
+      let!(:damage) { create :damage, damage_type: damage_type }
+      it 'has an ok status' do
+        submit
+        expect(response).to have_http_status :ok
+      end
+      it 'get the proper damage_type' do
+        submit
+        json = JSON.parse(response.body)
+        expect(json).to eql('uuid' => damage_type.uuid,
+                            'name' => damage_type.name,
+                            'damages' => [{ 'uuid' => damage.uuid }])
+      end
+    end
+    context 'damage_type not found' do
+      let(:damage_type) { double uuid: 0 }
+      it 'has a not found status' do
+        submit
+        expect(response).to have_http_status :not_found
+      end
+    end
+  end
+
+  describe 'POST #update' do
+    let(:submit) { post :update, id: damage_type.uuid, damage_type: changes }
+    let(:changes) { { name: 'New Damage Type Name' } }
+    context 'damage type found' do
+      let(:damage_type) { create :damage_type }
+      context 'changes apply successfully' do
+        it 'has a ok status' do
+          submit
+          expect(response).to have_http_status :ok
+        end
+        it 'calls #update on the item type with the changes' do
+          expect_any_instance_of(DamageType)
+            .to receive(:update)
+            .with(changes.stringify_keys)
+            .and_call_original
+          submit
+        end
+        it 'responds with the new attributes' do
+          submit
+          expect(response.body).to eql(
+            damage_type.reload.to_json(except:
+                                     %i(created_at updated_at creator_id id))
+          )
+        end
+      end
+      context 'changes not apply' do
+        let(:changes) { { name: nil } }
+        let(:error_messages) { ["Name can't be blank"] }
```

Comment:
Since you only use this once, seems unnecessary to abstract it into a `let` block